content/en/bug-report.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Bug reports
+sidebar: false
+---
+
+**Please help us by reporting any problems you find.**
+
+NumPy and SciPy libraries make use of GitHub to do project management.
```

Comment:
Why do we talk about NumPy in the following? IMO we should just talk about SciPy.

Reply:
Removed in a00bf11.